### PR TITLE
feat: Create model asynchronously

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Since some processes run in an atomic block, the creation must happen in a different thread that's why the AuditModel creation is moved to an asynchronous task.

### CODE REVIEW

File: eox_audit_model/models.py [hunk 1]
```diff
@@ -163,25 +164,18 @@ def execute_action(cls, action, method, parameters, notes=None):
             status = Status.FAIL
             raise error
         finally:
-            if getattr(settings, 'ALLOW_EOX_AUDIT_MODEL', False):
-                audit_register = cls.objects.create(
+            if getattr(settings, 'ALLOW_EOX_AUDIT_MODEL', True):
```
Comment (mariajgrimaldi) [inline eox_audit_model/models.py:167]: why true is the new default?

**Review by mariajgrimaldi — APPROVED** [review]

File: eox_audit_model/models.py (hunk 1 above)
Comment (andrey-canon) [reply eox_audit_model/models.py:167]: yes, it is, this is a different ticket https://edunext.atlassian.net/browse/PS2021-226

**Review by magajh — APPROVED** [review]
Nice!

File: eox_audit_model/tests/tests_tasks.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    def setUp(self):
+        """Setup common conditions for every test case"""
+        def valid_method(a, b, c, d):
+            """Execute a mathematic operation."""
+            LOG.info('This is an info message')
+            return (a + b + c) / d
+
+        self.valid_method = valid_method
+
+    def test_valid_method(self):
```
Comment (MoisesGSalas) [inline eox_audit_model/tests/tests_tasks.py:30]: can you add a test for a method that raises an exception?

Comment (andrey-canon) [reply eox_audit_model/tests/tests_tasks.py:30]: I have to change the name, the task doesn't execute any method

**Review by MoisesGSalas — APPROVED** [review]

File: eox_audit_model/tests/tests_tasks.py (hunk 2 above)
Comment (andrey-canon) [reply eox_audit_model/tests/tests_tasks.py]: and there is already a test for the exception case,  https://github.com/eduNEXT/eox-audit-model/blob/79e8668ee1bd524fa087e9061c5587763aa66f54/eox_audit_model/tests/tests_models.py#L223

**Pull request merged**